bundles/plugins/org.bonitasoft.studio.contract/src/org/bonitasoft/studio/contract/core/mapping/operation/initializer/NewBusinessObjectListInitializer.java

```diff
@@ -122,7 +122,8 @@ protected String iteratorName(final BusinessObject bo) {
     }
 
     protected void appendExistingBusinessObjects(final StringBuilder scriptBuilder, final BusinessObject businessObject) {
```

Comment:
Can you just check all Caller of this method to evaluate if we missed a use case?